examples/estimate-shipping-prices.php

```diff
@@ -0,0 +1,140 @@
+	
+// Example from_address array
+// The complete refence for the address object is available here: https://goshippo.com/docs/reference#addresses
+$from_address = array(
+    'object_purpose' => 'QUOTE',
+    'name' => 'Mr Hippo',
+    'company' => 'Shippo',
+    'street1' => '215 Clayton St.',
+    'city' => 'San Francisco',
+    'state' => 'CA',
+    'zip' => '94117',
+    'country' => 'US',
+    'phone' => '+1 555 341 9393',
+    'email' => 'mr-hippo@goshipppo.com',
+);
+
+// Parcel information array
+// The complete reference for parcel object is here: https://goshippo.com/docs/reference#parcels
+$parcel = array(
+    'length'=> '5',
+    'width'=> '5',
+    'height'=> '5',
+    'distance_unit'=> 'in',
+    'weight'=> '2',
+    'mass_unit'=> 'lb',
+);
+
+
+// Collect the shipments to each address
+$shipments = array();
+foreach ($destination_zip_codes as $zip_code)
+{
+    // Example to_address with the zip code
+    // The complete refence for the address object is available here: https://goshippo.com/docs/reference#addresses
+    $to_address = array(
+        'object_purpose' => 'QUOTE',
+        'country' => 'US',
+        'zip' => $zip_code,
+    );
+	
+    // For each destination address we now create a Shipment object.
+    // async=false indicates that the function will wait until all rates are generated before it returns.
+    // The reference for the shipment object is here: https://goshippo.com/docs/reference#shipments
+    // By default Shippo API operates on an async basis. You can read about our async flow here: https://goshippo.com/docs/async
+	$shipments[] = Shippo_Shipment::create(array(
+        'object_purpose'=> 'QUOTE',
+        'address_from'=> $from_address,
+        'address_to'=> $to_address,
+        'parcel'=> $parcel,
+        'async'=> false
+    ));
+}
+
+// Collect all shipments rates
+$all_rates = array();
+foreach ($shipments as $shipment) {
+    $all_rates = array_merge($all_rates, $shipment['rates_list']);
+}
+
+// This function takes a list of $rates, filters only those rates in
+// the $delivery_window, and returns the rates estimation
+function calculate_rates_estimation($rates, $delivery_window) {
+    // Filter rates by delivery window
+    $eligible_rates = array_filter(
+        $rates,
+        function($rate) use($delivery_window){
+            return $rate['days'] <= $delivery_window;
+        }
+    );
+
+    // Calcualte estimations on the eligible_rates
+    $min = $eligible_rates[0]['amount'];
+    $max = 0.0;
+    $sum = 0.0;
+    foreach ($eligible_rates as $rate) {
+        $amount = $rate['amount'];
+
+        $min = min($min, $amount);
+        $max = max($max, $amount);
+        $sum += $amount;
+    }
+
+    return array(
+        'delivery_window' => $delivery_window,
+        'min' => $min,
+        'max' => $max,
+        'average' => $sum / count($eligible_rates),
+    );
+}
+
+// Show estimations for each delivery window
+foreach ($delivery_windows as $delivery_window) {
```

Comment:
I ran this code and for some reason the Min does not print correctly. here's the output I get:

```
PHP Notice:  Undefined offset: 0 in /home/lucas/shippo-wrappers/php-sample-project/estimate.php on line 112
For a delivery window of 1 days:
	Min. costs: 
	Max. costs: 99.58
	Avg. costs: 36.642727272727
PHP Notice:  Undefined offset: 0 in /home/lucas/shippo-wrappers/php-sample-project/estimate.php on line 112
For a delivery window of 3 days:
	Min. costs: 
	Max. costs: 99.58
	Avg. costs: 32.368461538462
For a delivery window of 7 days:
	Min. costs: 5.81
	Max. costs: 99.58
	Avg. costs: 25.94347826087

```

Can you please investigate @martinrubi ?